grouper/ctl/main.py

```diff
@@ -66,12 +66,12 @@ def main(sys_argv=sys.argv, session=None):
         sa_log.setLevel(logging.INFO)
 
     usecase_factory = create_sql_usecase_factory(settings, session)
-    command_factory = CtlCommandFactory(usecase_factory)
+    command_factory = CtlCommandFactory(settings, usecase_factory)
```

Comment:
Could you put a comment on the call to `global_settings_from_config` above to note that in this case the settings are used both as the global singleton (for code outside grouper/ctl) and as an object that's passed around?